### ISSUE TITLE
Add Accessibility statement

### ISSUE THREAD

**Issue by markusjt** [cessda/cessda.cdc.versions#568]
Add a new page for Accessibility statement

- WCAG 2.0 level A and AA required by law in most of Europe (for sites created after 2014, more lax for sites from before that) and I think it's also the case in Norway
- Link should be in the footer at least but can also be in other places, e.g. About page
- Compliance status (WCAG 2.1 level A and AA)
  - Soon WCAG 2.2 but the changes in it are quite minor
- Information about known accessibility issues
- How to get in contact if finding an issue not listed here (I'd assume email or feedback form)
- Can also list things like screen readers that the site has been tested with

**Comment by matthew-morris-cessda** [comment]
@markusjt Any updates?

**Comment by markusjt** [comment]
> @markusjt Any updates?

Now it should be ready unless I missed something else (already made one corrective commit). Kristina made some fixes to the text and approved it a while ago. After that I removed the part about titles not always being descriptive because it seems they generally are and I just missed it while checking CDC for all the stuff in WCAG 2.1 A and AA the first time around.
